server.ts

```diff
@@ -57,6 +57,12 @@ app.engine('html', ngExpressEngine({
 app.set('view engine', 'html');
 app.set('views', join(DIST_FOLDER, 'browser'));
 
+let xFrameOptions = require('x-frame-options');
```

Comment:
`import` should be used instead of `require`; and this should be placed at the top of the files with the other imports.

Reply:
Done.